src/lib/Tab/LocationView/UrlsTab.php

```diff
@@ -10,29 +10,41 @@
 
 use eZ\Publish\API\Repository\URLAliasService;
 use eZ\Publish\API\Repository\Values\Content\Location;
+use EzSystems\EzPlatformAdminUi\Form\Data\Content\CustomUrl\CustomUrlAddData;
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
 use EzSystems\EzPlatformAdminUi\Tab\AbstractTab;
 use EzSystems\EzPlatformAdminUi\Tab\OrderedTabInterface;
+use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 use Twig\Environment;
 
 class UrlsTab extends AbstractTab implements OrderedTabInterface
 {
+    const URI_FRAGMENT = 'ez-tab-location-view-urls_tab';
```

Comment:
ez-tab-location-view-urls-tab

Reply:
This is however correct name of the tab. Should I change it here on in all others occurrences?

Reply:
Yeah don't mix `_` and `-` it looks whack.